docs/app/Examples/collections/Breadcrumb/BreadcrumbExamples.js

```diff
@@ -0,0 +1,60 @@
+import React, { Component } from 'react'
+import ComponentExample from 'docs/app/Components/ComponentDoc/ComponentExample'
+import ExampleSection from 'docs/app/Components/ComponentDoc/ExampleSection'
+
+export default class BreadcrumbExamples extends Component {
+  render() {
+    return (
+      <div>
+        <ExampleSection title='Types'>
+          <ComponentExample
+            title='Breadcrumb'
+            description='A standard breadcrumb'
+            examplePath='collections/Breadcrumb/Types/BreadcrumbExample'
+          />
+          <ComponentExample examplePath='collections/Breadcrumb/Types/BreadcrumbDividerExample' />
+          <ComponentExample examplePath='collections/Breadcrumb/Types/BreadcrumbPropExample' />
+        </ExampleSection>
+
+        <ExampleSection title='Content'>
+          <ComponentExample
+            title='Divider'
+            description='A breadcrumb can contain a divider to show the relationship between sections, this can be
```

Comment:
Strings in JS cannot span more than one line unless you use backticks, but this preserves whitespace including line breaks.  Suggest using an array and joining the lines:

```js
description={[
  'A breadcrumb can contain a divider to show the relationship between sections,',
  'this can be formatted as an icon or text.',
].join(' ')}
```